examples/python/visual_prompting/README.md

```diff
@@ -0,0 +1,30 @@
+# Segment Anything example
+This example demonstrates how to use a Python API implementation of Segment Anything pipeline inference:
+- Create encoder and decoder models
+- Create a visual prompter pipeline
+- Use points as prompts
+- Visualized result is saved to `sam_result.jpg`
+
+## Prerequisites
+Install Model API from source. Please refer to the main [README](../../../README.md) for details.
+
+## Run example
+To run the example, please execute the following command:
+```bash
+python run.py <path_to_image> <encoder_path> <decoder_path> <prompts>
+```
+where prompts are in X Y format.
+
+To run the pipeline out-of-the box you can download the test data by running the following command from the repo root:
+```bash
+python tests/python/accuracy/prepare_data.py -d data
```

Comment:
```suggestion
pip install httpx
python tests/python/accuracy/prepare_data.py -d data
```
It seems that httpx is necessary but it isn't installed when installing model_api

Reply:
Added